recipes/RNAtools/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "RNAtools" %}
+{% set version = "0.8.4" %}
+{% set sha256 = "6e6285c54bba3e492581c3b90b2dd86e1f2c2605c6298bee5280fdf0ce19a39b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - dot2ct = RNAtools.scripts.dot2ct:main
+    - colorRNA = RNAtools.scripts.colorRNA:main
+    - arcsRNA = RNAtools.scripts.arcsRNA:main
+    - cleanCT = RNAtools.scripts.cleanCT:main
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - cython
+    - numpy
+  run:
+    - python
+    - numpy
+    - pandas
+    - networkx
+    - matplotlib
+    - scikit-learn
+
+test:
+  imports:
+    - RNAtools
+    - RNAtools.graph
+  commands:
+    - dot2ct -h
+    - colorRNA -h
+    - arcsRNA -h
+    - cleanCT -h
+
+about:
+  home: http://github.com/grice/RNAtools
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
It looks like the license isn't included in the .tar.gz, so you will need to [package the license manually](https://conda-forge.org/docs/meta.html#packaging-the-license-manually).

Reply:
Thanks for the catch, @CurtLH. I have basically done a version bump which has the license file included.